src/shared/ethtool-util.h

```diff
@@ -20,14 +20,16 @@ typedef enum Duplex {
 
 typedef enum NetDevFeature {
         NET_DEV_FEAT_RX,
-        NET_DEV_FEAT_TX,
         NET_DEV_FEAT_GSO,
         NET_DEV_FEAT_GRO,
         NET_DEV_FEAT_GRO_HW,
         NET_DEV_FEAT_LRO,
         NET_DEV_FEAT_TSO,
         NET_DEV_FEAT_TSO6,
         _NET_DEV_FEAT_MAX,
+
+        NET_DEV_FEAT_TX = _NET_DEV_FEAT_MAX,
+        _NET_DEV_FEAT_MAX_ALL,
```

Comment:
Can you add a comment explaining why you do this?

And I think `_ALL_MAX` would be a better suffix than `_MAX_ALL`

Reply:
or maybe even _FULL_MAX or _TOTAL_MAX or so?